components/footer/Footer.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  FAUCETS,
+  FooterLinkInfo,
+  LEGAL,
+  NETWORKS,
+  RESOURCES,
+  SDKs,
+  SOLUTIONS,
+} from "./footerLinks";
+import { SOCIALS } from "./socialLinks";
+import {
+  Box,
+  ButtonGroup,
+  Container,
+  Flex,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react";
+import { Logo } from "components/logo";
+import { PRODUCTS } from "components/product-pages/common/nav/DesktopMenu";
+import {
+  Heading,
+  LinkButton,
+  TrackedIconButton,
+  TrackedLink,
+} from "tw-components";
+
+type FooterLinkGroupProps = {
+  heading: string;
+  links: FooterLinkInfo[];
+};
+
+function FooterLinkGroup(props: FooterLinkGroupProps) {
+  return (
+    <Stack spacing="4" minW="36" flex="1" mb={{ md: 12, base: 14 }}>
+      <Heading as="h5" size="label.lg">
+        {props.heading}
+      </Heading>
+      <Stack spacing="3" shouldWrapChildren>
+        {props.links.map((link) => (
+          <TrackedLink
+            isExternal={link.isExternal}
+            key={link.label}
+            href={link.link}
+            category="footer"
+            label={link.label}
+          >
+            {link.name}
+          </TrackedLink>
+        ))}
+      </Stack>
+    </Stack>
+  );
+}
+
+function FooterLinksGrid() {
+  return (
+    <Flex
+      mb={10}
+      direction={{ base: "column-reverse", md: "column", lg: "row" }}
+      gap={{ base: "12", md: "8" }}
+    >
+      <SimpleGrid columns={{ base: 2, lg: 4 }} spacing="8">
```

Comment:
```suggestion
      <SimpleGrid columns={{ base: 2, lg: 4 }} spacing={8}>
```

prefer number over string when possible

Reply:
@jnsdls Got it - Changing to number. 

( [it was already like that](https://github.com/thirdweb-dev/dashboard/blob/main/components/product-pages/homepage/Footer.tsx#L109-L111) - I have mostly kept the code as is )

Reply:
yep, since you're touching it anyways wanted to get it more in line with what we're doing now :)